CHANGELOG.md

```diff
@@ -1,409 +1,7 @@
 CHANGELOG
 =========
 
-## 2.27.0
+The CHANGELOG v2 of braintree-web has been moved to [braintree-web-bower repo](https://github.com/braintree/braintree-web-bower/tree/2.x). 
 
-* Bugfix - Fixes an issue that prevented PayPal flows from working within the iOS Google Search app
-* Hosted Fields
-  * Support for MasterCard 2-series BIN ranges
-* PayPal
-  * A new option, `intent`, has been added.
-  * Add PayPal Credit support
+For v3, switch to the [master branch](https://github.com/braintree/braintree-web/blob/master/CHANGELOG.md).
```

Comment:
I'd consider saying "For the latest version," so that this stays up to date. Lukewarm about this suggestion.